src/bufrlib.h.in

```diff
@@ -18,10 +18,10 @@
 #include <string.h>
 #include <ctype.h>
 
-#define NFILES @NFILES@ 
-#define MAXCD @MAXCD@
-#define MAXNC @MAXNC@
-#define MXNAF @MXNAF@
+#define NFILES @NFILES@ /**< ??? */
```

Comment:
@jbathegit what should be where the ??? are?

Actually this whole file needs to be documented - all pre-processor defines must be documented. ;-)

Reply:
I'll just push up a follow-on commit to add this in, rather than creating a separate branch.  But that will save you having to cut+paste.

Reply:
done